File: .sdk/tm/csharp/test/ExistsTest.cs
// BluefinTecsEcr SDK exists test.

using Xunit;

using BluefinTecsEcrSdk;

namespace BluefinTecsEcrSdk.Test;

public class ExistsTest
{
    [Fact]
    public void TestMode()
    {
        var testsdk = BluefinTecsEcrSDK.TestSDK(null, null);
        Assert.NotNull(testsdk);
    }
}

File: swift/Tests/BluefinTecsEcrSdkTests/ExistsTest.swift
// BluefinTecsEcr SDK exists test.

import XCTest

@testable import BluefinTecsEcrSdk

final class ExistsTest: XCTestCase {
  func testMode() {
    let testsdk = BluefinTecsEcrSDK.testSDK(nil, nil)
    XCTAssertEqual(testsdk.mode, "test")
  }
}
